Handler.go

```diff
@@ -16,10 +16,14 @@ type moveResponse struct {
 }
 
 type newGameResponse struct {
-	Full   bool
+	Full bool
 	Player int
 }
 
+type newBoard struct {
```

Comment:
Is this struct necessary? I cant think off the top of my head how you would do it differently, but I feel like there might be an even easier way to reset the board.

Reply:
potentially not sending the board as json and using raw data instead. The handler from my gba server is just

>buf, err := ioutil.ReadAll(r.Body)
>if err!=nil {log.Fatal("request",err)}
>ROM = buf

sends and receives a 1D byte array, that may of may not work for a 2D int array